libraries/androidutils/src/main/kotlin/io/element/android/libraries/androidutils/throttler/FirstThrottler.kt

```diff
@@ -38,4 +38,8 @@ class FirstThrottler(private val minimumInterval: Long = 800) {
         // Too early
         return CanHandleResult.No(minimumInterval - delaySinceLast)
     }
+
+    fun canHandleAsBoolean(): Boolean {
```

Comment:
I'd call this `canHandle` and the other one could be `getResult`?

Reply:
I have rewritten this class.